.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @C1KB-tensileai @FICU-tensileai @AKDO-slalom @THOA-slalom @GPEC-adhoc @TUAS-forpeople @BKY3-forpeople @VPAI-bixal @MIPI-forpeople @GHHZ-bixal @RIE9-adhoc
+* @C1KB-tensileai @FICU-tensileai @AKDO-slalom @THOA-slalom @GPEC-adhoc @TUAS-forpeople @BKY3-forpeople @VPAI-bixal @MIPI-forpeople @GHHZ-bixal @RIE9-adhoc @a-pfeiffer @XiaoBarry @alihashmi-wh @fumanjie
```

Comment:
@FICU-tensileai looks like we have some issues with the user names supplied for Xiao and Manjie.
Trying to tag them in this comment makes me believe Xiao should be `BRSC-Mathematica`. I think Manjie's user name is correct, but he might need to request access from Jose Alarcon (assuming he has the correct job codes?)